common/src/chain/transaction/input.rs

```diff
@@ -23,9 +25,13 @@ impl OutPoint {
     pub fn get_output_index(&self) -> u32 {
         self.index
     }
+
+    pub fn is_coinbase(&self) -> bool {
```

Comment:
We're still undecided whether we're gonna define the coinbase like this or we're gonna put it within the `consensus_data` of the block. But let's keep this for now.

Reply:
OK.